coil-sample/src/main/java/coil/sample/MainActivity.kt

```diff
@@ -10,6 +10,7 @@ import androidx.lifecycle.Observer
 import androidx.recyclerview.widget.RecyclerView
 import androidx.recyclerview.widget.StaggeredGridLayoutManager
 import coil.api.load
+import coil.transform.RoundedCornersTransformation
 
 class MainActivity : AppCompatActivity() {
```

Comment:
Please remove the changes to the sample app.

Reply:
removed